<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="workspaceSelector">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="experimentEditor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="globalBox">
          <property name="title">
           <string>Global Properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="GlobalPropertyWidget" name="globalProperties" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="universeBox">
          <property name="title">
           <string>Spaces and Data Directories</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="UniverseEditorWidget" name="universeEditor" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statsWidget">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="computeStatsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Compute Stats</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="statsView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="TaggingWidget" name="taggingWidget">
       <attribute name="title">
        <string>Identification</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
      <widget class="AntEditorWidget" name="shappingWidget">
       <attribute name="title">
        <string>Ant Editor</string>
       </attribute>
      </widget>
      <widget class="ZoningWidget" name="zoningWidget">
       <attribute name="title">
        <string>Zoning</string>
       </attribute>
      </widget>
      <widget class="UserDataWidget" name="userData">
       <attribute name="title">
        <string>Timed Data</string>
       </attribute>
      </widget>
      <widget class="VisualizationWidget" name="visualizeWidget">
       <attribute name="title">
        <string>Visualize</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentsProjects">
     <property name="title">
      <string>Recents Projects</string>
     </property>
     <addaction name="recentFile1"/>
     <addaction name="recentFile2"/>
     <addaction name="recentFile3"/>
     <addaction name="recentFile4"/>
     <addaction name="recentFile5"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentsProjects"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionShowLog"/>
    <addaction name="actionShowAntSelector"/>
    <addaction name="separator"/>
    <addaction name="actionCopyTimeFromFrame"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
    <addaction name="actionOnlineHelp"/>
   </widget>
   <widget class="QMenu" name="menuMove">
    <property name="title">
     <string>Move</string>
    </property>
    <addaction name="actionNextCloseUp"/>
    <addaction name="actionPreviousCloseUp"/>
    <addaction name="separator"/>
    <addaction name="actionNextTag"/>
    <addaction name="actionPreviousTag"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMove"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>318</width>
     <height>73</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Ant Selector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="AntListWidget" name="antList" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>275</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAddTrackingDataDir">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add Tracking Data Directory</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a New Tracking Data Directory. The current project must be saved. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="icon">
    <iconset theme="system-search">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Online Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="recentFile1">
   <property name="text">
    <string>File1</string>
   </property>
   <property name="toolTip">
    <string>Open Recent File 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+1</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="recentFile2">
   <property name="text">
    <string>File2</string>
   </property>
   <property name="toolTip">
    <string>Open Recent File 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+2</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="recentFile3">
   <property name="text">
    <string>File3</string>
   </property>
   <property name="toolTip">
    <string>Open Recent File 3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+3</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="recentFile4">
   <property name="text">
    <string>File4</string>
   </property>
   <property name="toolTip">
    <string>Open Recent File 4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+4</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="recentFile5">
   <property name="text">
    <string>File5</string>
   </property>
   <property name="toolTip">
    <string>Open Recent File 5</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+5</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="text">
    <string>Show Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionShowAntSelector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Ant Selector</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionNextCloseUp">
   <property name="text">
    <string>Next Close Up</string>
   </property>
   <property name="shortcut">
    <string>Shift+Down</string>
   </property>
  </action>
  <action name="actionPreviousCloseUp">
   <property name="text">
    <string>Previous Close Up</string>
   </property>
   <property name="shortcut">
    <string>Shift+Up</string>
   </property>
  </action>
  <action name="actionNextTag">
   <property name="text">
    <string>Next Tag</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionPreviousTag">
   <property name="text">
    <string>Previous Tag</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionCopyTimeFromFrame">
   <property name="text">
    <string>Copy current time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlobalPropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/GlobalPropertyWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UniverseEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/UniverseEditorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AntListWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/AntListWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaggingWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/TaggingWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AntEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/AntEditorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserDataWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/UserDataWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisualizationWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/VisualizationWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZoningWidget</class>
   <extends>QWidget</extends>
   <header location="global">fort/studio/widget/ZoningWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
